force-app/main/default/classes/CaseServiceTest.cls

```diff
@@ -0,0 +1,192 @@
+@isTest
+public with sharing class CaseServiceTest {
+    @isTest
+    public static void refuseCaseCreationWhenAnAccountIsNotAttachedToACase(){
+        // Given
+
+        Case request = new Case(Origin = 'Phone',Subject = 'Event Booking', Description = 'Book Spots');
```

Comment:
same here, better to use the setup method to insert data and u can use a  data factory framework to help you